sentry-android-integration-tests/sentry-uitest-android-benchmark/src/androidTest/java/io/sentry/uitest/android/benchmark/util/BenchmarkComparisonResult.kt

```diff
@@ -0,0 +1,153 @@
+package io.sentry.uitest.android.benchmark.util
+
+import java.util.concurrent.TimeUnit
+
+/** Result of the [BenchmarkOperation] comparison. */
+internal data class BenchmarkComparisonResult(
+    /** Number of measured iterations. */
+    val iterations: Int,
+    /** Screen refresh rate. */
+    val refreshRate: Float,
+    /** Screen refresh rate. */
+    val cores: Int,
+    /** Name of the first compared operation. */
+    val op1Name: String,
+    /** Name of the second compared operation. */
+    val op2Name: String,
+    /** Raw cpu time in milliseconds of op1. */
+    val op1CpuTime: List<Long>,
+    /** Raw cpu time in milliseconds of op2. */
+    val op2CpuTime: List<Long>,
+    /** Increase of cpu time in milliseconds. */
+    val cpuTimeIncreases: List<Long>,
+    /** Increase of cpu time in percentage. */
+    val cpuTimeIncreasePercentages: List<Double>,
+    /** Raw dropped frames of op1. */
+    val op1DroppedFrames: List<Double>,
+    /** Raw dropped frames of op2. */
+    val op2DroppedFrames: List<Double>,
+    /** Increase of dropped frames. */
+    val droppedFramesIncreases: List<Double>,
+    /** Increase of dropped frames in percentage. */
+    val droppedFramesIncreasePercentages: List<Double>,
+    /** Raw duration in nanoseconds of op1. */
+    val op1Duration: List<Long>,
+    /** Raw duration in nanoseconds of op2. */
+    val op2Duration: List<Long>,
+    /** Increase of duration in nanoseconds. If it's low enough, no end user will ever realize it. */
+    val durationIncreaseNanos: List<Long>,
+    /** Increase of duration in percentage. */
+    val durationIncreasePercentage: List<Double>,
+    /** Raw fps of op1. */
+    val op1Fps: List<Int>,
+    /** Raw fps of op2. */
+    val op2Fps: List<Int>,
+    /** Decrease of fps. */
+    val fpsDecreases: List<Int>,
+    /** Decrease of fps in percentage. */
+    val fpsDecreasePercentages: List<Double>
+) {
+
+    /**
+     * Prints the raw results of all runs of the comparison.
+     * Each printed line is prefixed by [prefix], to allow parsers to easily parse log files to read raw values.
+     */
+    fun printAllRuns(prefix: String) {
+        repeat(iterations) { index ->
+
+            println("$prefix ==================== Iteration $index ====================")
```

Comment:
Just for me to understand - this is gonna be printed into Logcat and later you pull this through adb probably, and parse, correct?